### PR TITLE
Implemented POST Operation via a Fake API

### PR DESCRIPTION
**Pull request by repo-son**
Implemented POST in Login form using a Fake API

### CODE REVIEW

**Review by sanchitha-prasad — COMMENTED** [review]
test review

File: TestLogin/TestLogin/MainPage.xaml [hunk 1]
```diff
@@ -62,7 +62,7 @@
         </Label>
 
         <Button x:Name="loginButton" Command="{Binding LoginCommand}" Text="Login" TextColor="Black" CornerRadius="65" BorderWidth="2" BorderColor="LimeGreen" BackgroundColor="White"
```
Comment (sanchitha-prasad) [inline TestLogin/TestLogin/MainPage.xaml]: i thing remove that x:name

Comment (repo-son) [reply TestLogin/TestLogin/MainPage.xaml]: Ok will do that

**Review by sanchitha-prasad — COMMENTED** [review]
All things up to date. Implementation is great. good work

**Pull request merged**